meteor/public/locales/nb/translations.json

```diff
@@ -934,8 +934,8 @@
     "Stop": "Stopp",
     "Clip \"{{fileName}}\" can't be played because it doesn't exist on the playout system": "Klippet \"{{fileName}}\" kan ikke spilles av fordi det ikke finnes på utspillingssystemet",
     "{{sourceLayer}} is not yet ready on the playout system": "{{sourceLayer}} er ennå ikke klar til å spilles ut fra avviklingsserver",
-    "{{sourceLayer}} is transferring to the the playout system": "{{sourceLayer}} overføres til avviklingsserver",
-    "{{sourceLayer}} is transferring to the the playout system and cannot be played yet": "{{sourceLayer}} overføres til avviklingsserver og kan ikke spilles av ennå",
+    "{{sourceLayer}} is transferring to  playout system": "{{sourceLayer}} overføres til avviklingsserver",
```

Comment:
I think this needs to be the same as line 936 in `nn/translations.json` and also needs to match line 55 in `lib/mediaObjects.ts`, otherwise it will just not work.

Reply:
OK, VisualCode auto-save problem. I must have committed the file before it auto-saved.